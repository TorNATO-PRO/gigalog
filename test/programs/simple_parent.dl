parent(john, mary).
parent(mary, bob).
parent(bob, alice).
