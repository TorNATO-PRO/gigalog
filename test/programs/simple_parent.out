parent(bob,alice)
parent(john,mary)
parent(mary,bob)
